### ISSUE TITLE
[V3] Help & set prefix

### ISSUE THREAD

**Issue by mikeshardmind** [Cog-Creators/Red-DiscordBot#1807]
The two bugs appear possibly related due to how pagination is handled in the helpformatter for embeds, but with permissions loaded on V3/develop, `[p]help` breaks at a field limit (which was previously adressed), and `[p]set prefix` breaks by missing coroutine on config value get.

```
discord.errors.HTTPException: BAD REQUEST (status code: 400): Invalid Form Body
In embed.fields.1.value: Must be 1024 or fewer in length.
  File "/home/michael/.local/share/virtualenvs/Red-DiscordBot-sAHpX_Uf/src/discord.py/discord/ext/commands/core.py", line 62, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "/home/michael/git/Red-DiscordBot/redbot/core/help_formatter.py", line 411, in help
    await destination.send(embed=embed)
  File "/home/michael/.local/share/virtualenvs/Red-DiscordBot-sAHpX_Uf/src/discord.py/discord/abc.py", line 769, in send
    data = yield from state.http.send_message(channel.id, content, tts=tts, embed=embed, nonce=nonce)
  File "/home/michael/.local/share/virtualenvs/Red-DiscordBot-sAHpX_Uf/src/discord.py/discord/http.py", line 216, in request
    raise HTTPException(r, data)
[06/06/2018 02:22] ERROR events on_command_error 204: Exception in command 'help'
Traceback (most recent call last):
  File "/home/michael/git/Red-DiscordBot/redbot/core/help_formatter.py", line 408, in help
    await destination.send(embed=embed)
  File "/home/michael/.local/share/virtualenvs/Red-DiscordBot-sAHpX_Uf/src/discord.py/discord/abc.py", line 769, in send
    data = yield from state.http.send_message(channel.id, content, tts=tts, embed=embed, nonce=nonce)
  File "/home/michael/.local/share/virtualenvs/Red-DiscordBot-sAHpX_Uf/src/discord.py/discord/http.py", line 216, in request
    raise HTTPException(r, data)
discord.errors.HTTPException: BAD REQUEST (status code: 400): Invalid Form Body
In embed.fields.1.value: Must be 1024 or fewer in length.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/michael/.local/share/virtualenvs/Red-DiscordBot-sAHpX_Uf/src/discord.py/discord/ext/commands/core.py", line 62, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "/home/michael/git/Red-DiscordBot/redbot/core/help_formatter.py", line 411, in help
    await destination.send(embed=embed)
  File "/home/michael/.local/share/virtualenvs/Red-DiscordBot-sAHpX_Uf/src/discord.py/discord/abc.py", line 769, in send
    data = yield from state.http.send_message(channel.id, content, tts=tts, embed=embed, nonce=nonce)
  File "/home/michael/.local/share/virtualenvs/Red-DiscordBot-sAHpX_Uf/src/discord.py/discord/http.py", line 216, in request
    raise HTTPException(r, data)
discord.errors.HTTPException: BAD REQUEST (status code: 400): Invalid Form Body
In embed.fields.1.value: Must be 1024 or fewer in length.
/home/michael/git/Red-DiscordBot/redbot/core/core_commands.py:860: RuntimeWarning: coroutine 'Value._get' was never awaited
  await self._prefixes(prefixes)
```

**Comment by mikeshardmind** [comment]
The two occured at the same time, but have seperate fixes involved. Fixing just the missing await on setting prefixes #1809 did not fix the pagination problem.

**Comment by mikeshardmind** [comment]
This can be closed when #1813 is merged.

**Comment by tekulvw** [comment]
Fixed with #1813
